src/admin/api.js

```diff
@@ -305,7 +338,28 @@ function buildCurrencyChannelRates(rates) {
     });
 }
 
+/**
+ * @method extractSourceCurrency
```

Comment:
This is actually just a function, are we able to identify it that way?

Reply:
Yes I just searched for it and there seems to be a `@function` directive, I thought there isn't. Will change it to that.